src/main/java/uk/gov/hmcts/reform/prl/services/noticeofchange/NoticeOfChangePartiesService.java

```diff
@@ -654,6 +658,9 @@ public Map<String, Object> aboutToSubmitStopRepresenting(String authorisation,
                     }
                 });
         }
+        caseAssignmentService.removeAmBarristerCaseRole(caseData,
```

Comment:
I think this is also behind the feature flag

Reply:
Flag check is present in the CaseAssignmentService removeAmBarristerIfPresent method